File: PicassoSample/app/src/main/java/com/github/ivanshafran/picassosample/MainActivity.java
package com.github.ivanshafran.picassosample;

import android.os.Bundle;
import android.widget.ImageView;

import com.squareup.picasso.MemoryPolicy;
import com.squareup.picasso.Picasso;

import androidx.appcompat.app.AppCompatActivity;

public class MainActivity extends AppCompatActivity {

	@Override
	protected void onCreate( Bundle savedInstanceState )
	{
		super.onCreate( savedInstanceState );
		setContentView( R.layout.activity_main );
		ImageView imageView = findViewById( R.id.imageView );

        Picasso
                .get()
                .load("https://i.pinimg.com/originals/e8/8f/30/e88f3028afe762960b7a2c11837b34d1.jpg")
                .fit()
                .centerInside()
                .into(imageView);
    }
}

File: NetSample/app/src/main/java/com/github/ivanshafran/netsample/MainActivity.kt
package com.github.ivanshafran.netsample

import android.content.Intent
import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.view.View
import com.github.ivanshafran.netsample.httpurlconnection.HttpUrlConnectionActivity
import com.github.ivanshafran.netsample.okhttp.OkHttpActivity
import com.github.ivanshafran.netsample.retrofit.RetrofitActivity

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        findViewById<View>(R.id.http_url_connection_button).setOnClickListener {
            startActivity(Intent(this, HttpUrlConnectionActivity::class.java))
        }

        findViewById<View>(R.id.okhttp_button).setOnClickListener {
            startActivity(Intent(this, OkHttpActivity::class.java))
        }

        findViewById<View>(R.id.retrofit_button).setOnClickListener {
            startActivity(Intent(this, RetrofitActivity::class.java))
        }
    }
}
